server/controllers/VerificationRequestController.ts

```diff
@@ -362,7 +369,10 @@ export class VerificationRequestController {
             ${applicationNames.map((a) => `<li>${a}</li>`).join('')}
             </ul>
           ` : ''}
-          <p>If you have further questions, please feel free to reach out to <a href="mailto:support@libretexts.org">support@libretexts.org</a>.</p>
+          ${adaptAccessCode ? `
+            <p><strong>Your ADAPT access code is: ${adaptAccessCode}</strong> . You can create your ADAPT instructor account <a href="https://adapt.libretexts.org/register/instructor" target="_blank">here</a>. Please note, access codes are only valid for 48 hours from their creation.</p>
+          ` : ''}
```

Comment:
Add a new line:
`<p><strong>Remember:</strong> ADAPT isn't integrated with LibreOne yet. You'll need to create separate credentials for use with ADAPT until that time.</p>`